napari/_qt/widgets/_tests/test_shortcut_editor_widget.py

```diff
@@ -45,3 +61,118 @@ def test_mark_conflicts(shortcut_editor_widget, qtbot):
     assert widget._mark_conflicts("Y", 1)
     # "Y" is arbitrary chosen and on conflict with existing shortcut should be changed
     qtbot.add_widget(widget._warn_dialog)
+
+
+def test_restore_defaults(shortcut_editor_widget, qtbot):
```

Comment:
`qtbot` is being used implicitly through the `shortcut_editor_widget`, right?

As it's not explicitly used here, I assume it's not doing anything.

```suggestion
def test_restore_defaults(shortcut_editor_widget):
```